__tests__/app.test.js

```diff
@@ -1,40 +1,28 @@
 const seed = require("../db/seeds/seed");
-const {
-  articleData,
-  commentData,
-  topicData,
-  userData,
-} = require("../db/data/test-data");
+const testData = require("../db/data/test-data");
 const db = require("../db/connection");
 const request = require("supertest");
 const app = require("../db/app");
 const sorted = require("jest-sorted");
 
-beforeEach(() => seed({ articleData, commentData, topicData, userData }));
-// afterAll(() => db.end);
-
-// describe.only("Name of the group", () => {
-//   test("should ", () => {
-//     return db.query("SELECT * FROM topics").then((topics) => {
-//       console.log(topics.rows);
-//     });
-//   });
-// });
+beforeEach(() => seed(testData));
+afterAll(() => db.end);
 
 describe("News Express App", () => {
-  describe("get/api/topics", () => {
+  describe("GET api/topics", () => {
     test("200: responds with array of objects, each of which have properties slug & description ", () => {
       return request(app)
         .get("/api/topics")
         .expect(200)
         .then(({ body }) => {
-          body.forEach((topic) => {
+          expect(body.topics).toHaveLength(3);
+          body.topics.forEach((topic) => {
             expect(topic).toHaveProperty("slug");
             expect(topic).toHaveProperty("description");
           });
         });
     });
-    test("404: responds with a message when non existant endpoint requested ", () => {
+    test("404: responds with a message when article ID of invalid number endpoint requested ", () => {
```

Comment:
this message for this test is inaccurate - the error you're checking is an invalid path, it's not specific to article ID. I'd  change that probably back to the original test description